kasa/smart/smartdevice.py

```diff
@@ -309,6 +308,16 @@ async def _initialize_features(self):
             for feat in module._module_features.values():
                 self._add_feature(feat)
 
+    def get_module(self, module_name) -> SmartModule | None:
+        """Return the module from the device modules or None if not present."""
+        if module_name in self.modules:
+            return self.modules[module_name]
+        elif self._exposes_child_modules:
+            for child in self._children.values():
+                if module_name in child.modules:
+                    return child.modules[module_name]
+        return None
```

Comment:
Probably a good idea to add a test for this behavior, too.

Reply:
Done